### PR TITLE
NAS-111163 / 21.08 / Add revoke cert and CA option to tables

### PR DESCRIPTION
**Pull request by RehanY147**
NAS-111163

### CODE REVIEW

File: src/app/pages/credentials/certificates-dash/certificates-dash.component.ts [hunk 1]
```diff
@@ -103,6 +111,17 @@ export class CertificatesDashComponent implements OnInit {
             {
               name: T('CN'), prop1: 'common', name2: T('SAN'), prop2: 'san',
             },
+            {
+              name: T('Status'),
+              prop1: 'revoked',
+              matTooltip: 'Revoked',
```
Comment (denysbutenko) [inline src/app/pages/credentials/certificates-dash/certificates-dash.component.ts:117]: The string should have i18n support

File: src/app/pages/credentials/certificates-dash/certificates-dash.component.ts [hunk 2]
```diff
@@ -159,6 +178,17 @@ export class CertificatesDashComponent implements OnInit {
...
```
Comment (denysbutenko) [inline src/app/pages/credentials/certificates-dash/certificates-dash.component.ts:184]: same

File: src/app/pages/credentials/certificates-dash/certificates-dash.component.ts [hunk 3]
```diff
@@ -249,52 +279,80 @@ export class CertificatesDashComponent implements OnInit {
...
-          },
-          (err) => {
-            new EntityUtils().handleWSError(this, err, this.dialogService);
-          },
-        );
-        event.stopPropagation();
+    this.downloadActions = [
+      {
+        icon: 'save_alt',
+        name: 'download',
```
Comment (denysbutenko) [inline src/app/pages/credentials/certificates-dash/certificates-dash.component.ts:285]: This is probably out of scope. Would you add matTooltip for the download button, please?

https://user-images.githubusercontent.com/351613/127840957-730665fa-c468-4f4a-8f09-ffb0dfc5984f.mp4

Comment (RehanY147) [reply src/app/pages/credentials/certificates-dash/certificates-dash.component.ts:285]: Sure :)

**Pull request merged**